docs/classifier_example/classification_v4_4_features.ipynb

```diff
@@ -0,0 +1,884 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[-0.32695550305984783,\n",
+       " 0.4736868636179572,\n",
+       " -0.5699845991812187,\n",
+       " -0.5863773622428564]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "iris_setosa_normalised, iris_versicolor_normalised = DataPlotter.load_data(max_features=4)  # Load scaled and normalised data\n",
+    "first_data_point = [feature[0] for feature in iris_setosa_normalised]\n",
+    "first_data_point"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We want to build the circuit such that:\n",
+    "\\begin{equation}\\ket{00} \\mapsto -0.3270 \\ket{00} + 0.4737 \\ket{01} - 0.5700 \\ket{10} - 0.5864 \\ket{11}\\end{equation}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a00, a01, a10, a11 = first_data_point\n",
+    "alpha1 = atan(sqrt((a10**2  + a11**2) / (a00**2  + a01**2)))\n",
+    "alpha2 = np.arctan2(a01, a00) \n",
+    "alpha3 = np.arctan2(a11, a10) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As always the first step is to set up a connection with the Quantum Inspire:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter email:\n",
+      "demo@quantum-inspire.com\n",
+      "Enter password\n",
+      "········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from getpass import getpass\n",
+    "from coreapi.auth import BasicAuthentication\n",
+    "\n",
+    "from qiskit.validation.base import Obj\n",
+    "from qiskit.circuit import QuantumRegister, ClassicalRegister, QuantumCircuit\n",
+    "from qiskit.tools.visualization import plot_state, plot_histogram, circuit_drawer\n",
+    "from qiskit import execute\n",
+    "\n",
+    "from quantuminspire.qiskit import QI\n",
+    "from quantuminspire.api import QuantumInspireAPI\n",
+    "\n",
+    "\n",
+    "def get_authentication():\n",
+    "    \"\"\" Gets the authentication for connecting to the Quantum Inspire API.\"\"\"\n",
+    "    if 'password' not in vars().keys():\n",
+    "        print('Enter email:')\n",
+    "        email = input()\n",
+    "        print('Enter password')\n",
+    "        password = getpass()\n",
+    "    return BasicAuthentication(email, password)\n",
+    "\n",
+    "authentication = get_authentication()\n",
```

Comment:
Same comments as above, and also holds for the first notebook

Reply:
Adjusted